aws-test/tests/aws_cloudfront_function/variables.tf

```diff
@@ -0,0 +1,82 @@
+variable "resource_name" {
+  type        = string
+  default     = "turbot_test_20200125_create_update"
+  description = "Name of the resource used throughout the test."
+}
+
+variable "aws_profile" {
+  type        = string
+  default     = "integration-tests"
```

Comment:
```suggestion
  default     = "default"
```

Reply:
@omerosaienni Please keep the test profile as the default, for all other tables we are making the changes.